man/systemd-cryptenroll.xml

```diff
@@ -409,11 +409,34 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-external-key=</option><replaceable>PATH</replaceable></term>
+
+        <listitem><para>Enroll an external TPM2 security chip. Expects a path referring to the TPM2 public
```

Comment:
maybe say what "external" is supposed to mean. Maybe should use words like "offline"?

Reply:
is the word "external" the terminology used elsewhere for this concept?

otherwise maybe call this an "offline key" or "disconnected key" or so?

Reply:
Good point, how about calling the parameter `--tpm2-device-key=` so it matches up with the `--tpm2-device=` parameter? I reworded the man page also to indicate it performs the same operation as `--tpm2-device=` but using calculation, instead of direct access to the tpm.